src/analyses/mCP.ml

```diff
@@ -76,11 +82,16 @@ struct
       List.map f
     in
     let xs = get_string_list "ana.activated" in
+    let special_inse = specific_cont_sens xs in
     let xs = map' find_id xs in
     base_id := find_id "base";
     activated := map (fun s -> s, find_spec s) xs;
-    path_sens := map' find_id @@ get_string_list "ana.path_sens";
-    cont_inse := map' find_id @@ get_string_list "ana.ctx_insens";
+    (*checks if an analysis is enabled with special handled context sensitivity*)
+    if Option.is_some special_inse
+    then (cont_inse := map' find_id (Option.get special_inse);
+          path_sens := [])
+    else (cont_inse := map' find_id @@ get_string_list "ana.ctx_insens";
+          path_sens := map' find_id @@ get_string_list "ana.path_sens");
```

Comment:
If you rewrite this as a `match` that will probably be nicer. Also, why does this set all analyses to not be path-sensitive?

Reply:
This comes from a recent unsound issue we discovered (see Issue #1302). When analyzing a function that is called with 2 different values, 2 different paths and the same context, the values are merged and can lead to "bottom" in the apron analysis. 
Disabeling the path sensitivity was a "workaround" for this problem, but @jerhard and I are currently trying to fix this.

Reply:
We could detect the issue, and revert this change (so now the call-string analysis and the loopfree-callstring analysis are path-sensitive).
See  #1354